dev-scripts/make-release.sh

```diff
@@ -48,7 +47,7 @@ INSTALLER_ETC_DIR="$INSTALLER_PKG_DIR/etc"
 INSTALLER_COMPOSE_DIR="$INSTALLER_LIB_DIR/docker-compose"
 INSTALLER_MAIN_SCRIPT="$INSTALLER_BIN_DIR/ipfix-rita"
 
-INSTALLER_INSTALL_SCRIPT="$INSTALLER_DIR/install-ipfix-rita.sh"
+INSTALLER_SCRIPTS_DIR=$"../install-scripts"
```

Comment:
Since we're bundling the docker scripts and whatnot, it makes sense to move to an install scripts directory.